boost-maven/boost-maven-plugin/src/main/java/io/openliberty/boost/maven/utils/MavenProjectUtil.java

```diff
@@ -56,5 +58,22 @@ public static String findSpringBootVersion(MavenProject project) {
 
         return springFrameworkDependencies;
     }
+    
+    public static Map<String, String> getBoosterDependencies(MavenProject project, BoostLogger logger) {
+
+        Map<String, String> boosterDependencies = new HashMap<String, String>();
+        logger.debug("Processing project for dependencies.");
+
+        for (Artifact artifact : project.getArtifacts()) {
+            logger.debug("Found dependency while processing project: " + artifact.getGroupId() + ":"
+                    + artifact.getArtifactId() + ":" + artifact.getVersion());
+            
+            if (artifact.getGroupId().equals("io.openliberty.boosters")) {
```

Comment:
Should this be a string constant somewhere (not sure where)?

Reply:
Agree. I will change and test this in the next item